src/main/java/co/cask/gcp/bigquery/BigQuerySource.java

```diff
@@ -164,6 +169,22 @@ public String getInputFormatClassName() {
         return config;
       }
     }));
+
+    if (config.schema != null) {
+      try {
+        Schema schema = Schema.parseJson(config.schema);
+        if (schema.getFields() != null) {
+          FieldOperation operation =
+            new FieldReadOperation("Read", "Read from BigQuery table",
+                                   EndPoint.of(context.getNamespace(), config.table),
```

Comment:
these plugins should be using a reference name. Is that coming in a separate PR?